client/my-sites/people/followers-list/index.jsx

```diff
@@ -162,7 +168,29 @@ const Followers = localize(
 				return <EmptyContent title={ emptyTitle } />;
 			}
 
-			let headerText = this.props.label;
+			let headerText;
+			if ( this.props.totalFollowers ) {
+				headerText = this.props.translate(
+					'You have %(number)d follower',
+					'You have %(number)d followers',
+					{
+						args: { number: this.props.totalFollowers },
+						count: this.props.totalUsers,
```

Comment:
Should this be `this.props.totalFollowers`? In testing I'm seeing `You have 1 followers` rather than `You have 1 follower` and I think this might be why.

Reply:
Ooops, good point, fixed 👍